package-lock.json

```diff
@@ -1,11827 +0,0 @@
-{
```

Comment:
Shouldn't package-lock.json remain? Otherwise our builds won't be repeatable since the precise versions of the packages used won't be the same.

Reply:
Because we need to use yarn and not npm I removed the npm `package-lock.json` file, it can also not be uploaded because it's in the `.gitignore` file